common/src/chain/config/mod.rs

```diff
@@ -567,30 +568,51 @@ fn create_testnet_genesis() -> Genesis {
     )
 }
 
-pub fn regtest_genesis_values() -> (
+pub fn regtest_genesis_values(
+    genesis_staking_settings: Option<String>,
+) -> (
```

Comment:
Would be nicer for this to take some sort of setting type with `Default` impl to avoid the weird special casing for `None`

Reply:
Done